### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by amitava82** [cmswalker/fullpage-react#45]
Just installed from npm and trying to setup basic example, it throws `Uncaught (in promise) Error: t.render(): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.`

```
const { Fullpage, Slide}  = require('fullpage-react');
    const option = {
      // for mouse/wheel events
      // represents the level of force required to generate a slide change on non-mobile, 10 is default
      scrollSensitivity: 7,
      // for touchStart/touchEnd/mobile scrolling
      // represents the level of force required to generate a slide change on mobile, 10 is default
      touchSensitivity: 7,
      scrollSpeed: 500,
      hideScrollBars: true,
      enableArrowKeys: false,
      breakpoint: 375 // Disable FullpageReact and get standard scroll behavior back when this breakpoint (or below) is reached
    };

    const verticalSlides = [
      <Slide style={{backgroundColor: 'blue'}}>
        <p>Slide 1</p>
      </Slide>,
      <Slide style={{backgroundColor: 'pink'}}><p>Slide 2</p></Slide>
    ];

    option.slides = verticalSlides;

    return <Fullpage {...option} />;

**Comment by amitava82** [comment]
OK, looks like `Fullpage` component needs to have a children which should not be required.

**Comment by cmswalker** [comment]
Thanks for this. Fixed in [3.0.4](https://www.npmjs.com/package/fullpage-react)
